src/network/send.ts

```diff
@@ -204,6 +205,23 @@ export async function sendMessage({
     // console.log("=> realSatsContactId", realSatsContactId, contactId)
     if (isTribeOwner && amount && realSatsContactId === contactId) {
       mqttTopic = '' // FORCE KEYSEND!!!
+      try {
+        const receiver = (await models.ChatMember.findOne({
+          where: {
+            contactId: contactId,
+            tenant,
+            chatId: chat.id!,
+          },
+        })) as ChatMemberRecord
+        await receiver?.update({
+          totalEarned: receiver.totalEarned + amount,
+        })
+      } catch (error) {
+        sphinxLogger.error(
+          `=> Could not update ChatMember table for Leadership board`,
+          error
+        )
```

Comment:
Similar to above
sphinxLogger will get confused by that formatting

try something like this
```
sphinxLogger.error(`=> Could not update ChatMember table for Leadership board ${error}`, logging.Network)
```